### PR TITLE
Rely on `performance` from Polyfill.io in user-timings

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

`window.performance.now` is available since #24210, so we can take advantage of this fact and drop the requirement of `user-timings.js`.

We can also remove the commercial marks which we’re not currently using in `on-slot-render`, as we already capture `adOnPage`.

## Does this change need to be reproduced in dotcom-rendering ?

- [X] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## What is the value of this and can you measure success?

Smaller bundle, less dependencies.

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/dfp/on-slot-render.ts [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
 import reportError from '../../../../lib/report-error';
-import { markTime } from '../../../../lib/user-timing';
 import { dfpEnv } from './dfp-env';
 import { emptyAdvert } from './empty-advert';
 import { getAdvertById } from './get-advert-by-id';
 import { renderAdvert } from './render-advert';
 
-const recordFirstAdRendered = once(() => {
-	markTime('Commercial: First Ad Rendered');
-});
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/dfp/on-slot-render.ts:15]: Gone

File: static/src/javascripts/projects/common/modules/analytics/google.ts [hunk 2]
```diff
@@ -130,32 +130,29 @@ const trackPerformance = (
...
+	const timeSincePageLoad = Math.round(window.performance.now());
+	const event: TimingEvent = {
+		timingCategory,
+		timingVar,
+		timeSincePageLoad,
+		timingLabel,
+	};
+
 	// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- extra safety if undefined
-	if (window.performance?.now) {
```
Comment (mxdvl) [inline static/src/javascripts/projects/common/modules/analytics/google.ts:134]: Useless check, if we have the polyfill.

File: static/src/javascripts/projects/commercial/modules/dfp/load-advert.ts [hunk 3]
```diff
@@ -38,7 +37,6 @@ export const loadAdvert = (advert: Advert): void => {
 			// The display needs to be called, even in the event of an error.
 		})
 		.then(() => {
-			markTime(`Commercial: Slot Ready: ${advert.id}`);
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/dfp/load-advert.ts:41]: Redundant with `load-advert.ts` : https://github.com/guardian/frontend/blob/8e65a493564069661d2d4050c77f2933c9a0666e/static/src/javascripts/projects/commercial/modules/dfp/load-advert.ts#L40

File: tools/__tasks__/commercial/graph/output/standalone.commercial.ts.json [hunk 4]
```diff
@@ -219,11 +213,10 @@
  "../projects/commercial/modules/dfp/on-slot-render.ts": [
   "../../../../node_modules/@guardian/commercial-core/dist/cjs/index.d.ts",
   "../../../../node_modules/@guardian/commercial-core/dist/cjs/index.d.ts",
-  "../../../../node_modules/@types/lodash-es/index.d.ts",
```
Comment (mxdvl) [inline tools/__tasks__/commercial/graph/output/standalone.commercial.ts.json:222]: `isString` from our own lib.

**Comment by mchv** [comment]
Should `getCurrentTime()` from `user-timing.js` not be removed as well

**Review by arelra — APPROVED** [review]
Thanks for the run through 👍

**Pull request merged**